src/app/login/login.component.html

```diff
@@ -0,0 +1,33 @@
+<section id="testimonial" class="section-padding">
+    <div class="container">
+        <div class="buffer">
+        </div>
+        <div class="row">
+            <div class="col-md-4">
+            </div>
+            <div class="col-md-4">
+                <form role="form" action="javascript:;" method="post" id="login-form" autocomplete="off">
```

Comment:
`action="javascript:;" method="post"`

I believe we need to convert this into an `ngForm`

Reply:
sure i will, in my next commit.